diff --git a/node_modules/@radix-ui/react-use-effect-event/src/use-effect-event.tsx b/node_modules/@radix-ui/react-use-effect-event/src/use-effect-event.tsx
index 817c44d..52a980a 100644
--- a/node_modules/@radix-ui/react-use-effect-event/src/use-effect-event.tsx
+++ b/node_modules/@radix-ui/react-use-effect-event/src/use-effect-event.tsx
@@ -1,6 +1,6 @@
 /* eslint-disable react-hooks/rules-of-hooks */
 import { useLayoutEffect } from '@radix-ui/react-use-layout-effect';
-import * as React from 'react';
+import React, {experimental_useEffectEvent} from 'react';
 
 type AnyFunction = (...args: any[]) => any;
 
@@ -12,7 +12,7 @@ export function useEffectEvent<T extends AnyFunction>(callback?: T): T {
   // See https://github.com/webpack/webpack/issues/14814
   if (typeof (React as any)[new OpaqueString('useEffectEvent').toString()] === 'function') {
     // @ts-expect-error
-    return React.useEffectEvent(callback);
+    return experimental_useEffectEvent(callback);
   }
 
   const ref = React.useRef<AnyFunction | undefined>(() => {
